README.md

```diff
@@ -21,16 +21,17 @@ I've put together this project to help myself and others setup new web applicati
 ### Clone and install dependencies
 
 ```sh
-git clone https://github.com/architect-team/template-nuxt-tsx.git
-npm install
+$ git clone https://github.com/architect-templates/nuxt.git
+$ cd ./nuxt
+$ npm install
```

Comment:
I know you didn't actually add this, but I don't see a reason why installing dependencies is necessary here. This block would make more sense if it was structured like this one in the nestjs repo

https://github.com/architect-templates/nestjs#running-locally

Reply:
Good point, I was wondering if there was just some reason I wasn't aware of that `npm install` was being run as part of this - will restructure bits of this now so it looks more like the other READMEs